web/src/containers/activity/EntryDetailsDialog.js

```diff
@@ -0,0 +1,75 @@
+import { Dialog, Tabs, TabPanel } from '@cmsgov/design-system-core';
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import ContractorResources from './ContractorResources';
+import CostAllocate from './CostAllocate';
+import Costs from './Costs';
+import Overview from './Overview';
+import Goals from './Goals';
+import Schedule from './Schedule';
+import StandardsAndConditions from './StandardsAndConditions';
+
+const ActivityDialog = props => {
+  const { title, activityIndex, closeModal } = props;
+  const [showModal, setShowModal] = useState(true);
+
+  const handleOnExit = () => {
+    setShowModal(false);
+    closeModal();
+  };
+
+  return (
+    showModal && (
+      <Dialog
+        ariaCloseLabel={`Close modal for ${title}`}
+        className="ds-c-dialog--full"
+        closeButtonVariation="transparent"
+        closeText="Save and close"
+        onExit={() => handleOnExit()}
+        title={title}
+      >
+        <Tabs>
+          <TabPanel
+            id={`#activity-overview-${activityIndex}`}
```

Comment:
I thought I needed those to properly render the `href` for the tab, but it turns out I don't. I removed the pound and suffixed the IDs with `-tab` to prevent conflicts.